extensions/amp-analytics/0.1/test/test-instrumentation.js

```diff
@@ -87,4 +87,22 @@ describe('instrumentation', function() {
     expect(fnIdY.callCount).to.equal(1);
   });
 
+  it('should listen on custom events', () => {
```

Comment:
Was gonna document each specific custom event type in the upcoming PRs. Otherwise this is just an infrastructural change with no config consequences yet.